tests/test__ped_partition.py

```diff
@@ -177,18 +182,39 @@ def runTest(self):
         self._part.set_name("blah")
         self.assertEqual(self._part.get_name(), "blah")
 
-@unittest.skip("Unimplemented test case.")
-class PartitionIsBusyTestCase(unittest.TestCase):
+class PartitionIsBusyTestCase(RequiresPartition):
+    def setUp(self):
+        RequiresPartition.setUp(self)
+        self.mountpoint = None
+
+    def doMount(self):
+        self.mountpoint = tempfile.mkdtemp()
+        os.system("mount -o loop %s %s" % (self.path, self.mountpoint))
+
+    def mkfs(self):
+        os.system("/sbin/mkfs.ext2 -F -q %s" % self.path)
```

Comment:
I prefer to avoid explicit paths in os.system() calls.  It's really just a cosmetic thing, but arguably, you should have these tools in your PATH if you are working on pyparted.